docs/reference/commandline/manifest.md

```diff
@@ -0,0 +1,277 @@
+---
+title: "manifest"
+description: "The manifest command description and usage"
+keywords: "docker, manifest"
+---
+
+<!-- This file is maintained within the docker/cli Github
+     repository at https://github.com/docker/cli/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# manifest
+
+```markdown
+Usage:  docker manifest COMMAND
+
+Manage Docker image manifests and manifest lists
+
+Options:
+      --help   Print usage
+
+Commands:
+  annotate    Add additional information to a local image manifest
+  create      Create a local manifest list for annotating and pushing to a registry
+  inspect     Display an image manifest, or manifest list
+  push        Push a manifest list to a repository
+
+```
+
+## Description
+
+The `docker manifest` command by itself performs no action. In order to operate
+on a `manifest` or `manifest list`, one of the subcommands must be used.
```

Comment:
should `manifest` and `manifest list` be quoted instead of backticked?

Reply:
If you are thinking of them as new terms, use italic (but you need to define them). Otherwise, they don't really need any mark-up.